README.rst

```diff
@@ -23,20 +23,24 @@
 .. _Gitter: https://gitter.im/matplotlib/matplotlib
 
 .. |NUMFocus| image:: https://img.shields.io/badge/powered%20by-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A
-.. _NUMFocus: http://www.numfocus.org
+.. _NUMFocus: https://www.numfocus.org
 
+<<<<<<< HEAD
```

Comment:
There‘s still diff information in your commit.

Reply:
🤦‍♀️that wasn't how it looked on my local machine